packages/block-library/src/column/edit.js

```diff
@@ -61,11 +64,14 @@ function ColumnEdit( {
 				templateLock={ false }
 				renderAppender={
 					hasChildBlocks
-						? undefined
+						? false
 						: () => <InnerBlocks.ButtonBlockAppender />
 				}
+				tagName={ Block.div }
+				className={ classes }
+				style={ hasWidth ? { flexBasis: width + '%' } : undefined }
```

Comment:
I'm wonder if instead of having these extra props here we ccould have a way to include these in the tagName

```
function Column = ( props ) => <Block.div {...props } className={ classnames( props.className, classes ) } style={ someStyle } /> 
```

And use `Column` as a tag name. 

This could be a way to limit the public API. Thoughts? too hard to understand?

Reply:
That could work. Downside is that this component must be created on every render, but that's probably fine if it's simple.

Reply:
Thinking about it more, I don't like that it's now the responsibility of the block to pass `props` and to make sure that these props are properly merged. If we need to isolate these props, perhaps it's better to nest them in an object. I don't have anything against that, but I don't have anything against not nesting them either. What do you think?

Reply:
Would you mind sharing a code example to clarify?

Reply:
For nesting props?

```
<InnerBlocks
	templateLock={ false }
	renderAppender={
		hasChildBlocks
			? false
			: () => <InnerBlocks.ButtonBlockAppender />
	}
	tagName={ Block.div }
	passedProps={ {
		className: classes,
		style: hasWidth ? { flexBasis: width + '%' } : undefined,
	} }
/>
```

Reply:
Btw, we have a similar situation for `RichText`. Probably would be good to stay consistent.

Reply:
Right, how do we do with RichText, just spread props?

Reply:
Yeah, we spread just like this PR atm.

Reply:
@youknowriad What do you think? Spread, nested object, or pass component?

Reply:
Honestly, I don't know yet :P Maybe one of "pass component" or "experimental nested object" to avoid having to make the decision now and explore more blocks first.

Reply:
Agreed.

Reply:
I think I'll go for experimentally nesting the props in an object. A new component becomes quite complex as you'd also have to forward the ref. :/